src/Pact/Native/Db.hs

```diff
@@ -220,6 +237,38 @@ read' g0 i as@(table@TTable {}:TLitString key:rest) = do
 
 read' _ i as = argsError i as
 
+
+foldDB' :: NativeFun e
+foldDB' i [tbl, TApp qry _, TApp consumer _] = do
+  table <- reduce tbl >>= \case
```

Comment:
On second thought let's go ahead and add `GFoldDb` as it gives the container (chainweb) optionality to charge more to address unforeseen issues. It can be nominal for now, e.g. "select without column filter" cost

Reply:
Done :+1: